vulcan/lib/server/workflows/scripts/dge_filter.py

```diff
@@ -21,12 +21,14 @@
 # min_pct
 min_pct = float(input_var('min_pct'))
 # Algorithm choice: OR filter, a.k.a. either group meets the cutoff
-pct_cols = [s for s in DEdf.keys() if (s=="pts" or s.startswith("pts_"))]
-pct_col1 = list(DEdf[pct_cols[0]])
-pct_col2 = list(DEdf[pct_cols[1]])
-idx_passed = [
-    i for i in list(DEdf_filt.index) if ( pct_col1[i] >= min_pct or pct_col2[i] >= min_pct )]
-DEdf_filt = DEdf_filt.loc[idx_passed,:]
+pct_cols = [s for s in DEdf_filt.keys() if (s=="pts" or s.startswith("pts_"))]
+pct_col1 = list(DEdf_filt[pct_cols[0]])
+pct_col2 = list(DEdf_filt[pct_cols[1]])
+pct_passed = list(map(
+    lambda i: (pct_col1[i] >= min_pct or pct_col2[i] >= min_pct),
+    list(DEdf_filt.index)
+))
+DEdf_filt = DEdf_filt.loc[pct_passed]
```

Comment:
Now I create a boolean list that's the same length as there are rows in the df, then use this variable with the same subsetting method as in all the other subsetting lines above.